proof/refine/ARM/IpcCancel_R.thy

```diff
@@ -105,10 +105,10 @@ lemma cancelSignal_st_tcb_at':
                     hoare_drop_imp[where R="\<lambda>rv _. \<exists>a b. delete t (f rv) = a # b" for f])
   done
 
-lemma cancelIPC_simple'_not_awaiting_reply:
+lemma cancelIPC_simple':
   "\<lbrace>\<lambda>s. sym_refs (state_refs_of' s)\<rbrace>
    cancelIPC t
-   \<lbrace>\<lambda>_. st_tcb_at' ((=) Running or (=) Inactive or (=) Restart or (=) IdleThreadState) t\<rbrace>"
```

Comment:
Ah, right, this is the result of the previous discussion. And now we are aligning `simple’` with it, which is good!